app/commands/todo/abandon.rb

```diff
@@ -7,12 +7,17 @@ module Abandon
         class Command
           attr_reader :payload, :aggregate_id
 
+          def self.build(**args)
```

Comment:
this method is duplicated in all the command classes? Shall we do something about that? Like introduce a base class for commands?